CMake/DisplayCluster.cmake

```diff
@@ -13,8 +13,11 @@ set(DISPLAYCLUSTER_DEB_DEPENDS libavutil-dev libavformat-dev libavcodec-dev
   libopenmpi-dev openmpi-bin libswscale-dev libxmu-dev libpoppler-qt5-dev
   libboost-date-time-dev libboost-serialization-dev libboost-test-dev
   libboost-program-options-dev libboost-regex-dev libboost-system-dev
-  libboost-thread-dev libfcgi-dev)
-set(DISPLAYCLUSTER_PORT_DEPENDS ffmpeg boost poppler fcgi)
+  libboost-thread-dev libfcgi-dev qt5-default libqt5core5 libqt5declarative5
+  libqt5script5 libqt5scripttools5 libqt5svg5-dev libqt5webkit5-dev
+  libqt5xmlpatterns5-dev libqt5x11extras5-dev qtquick1-5-dev qtscript5-dev
+  libpoppler-qt5-dev)
+set(DISPLAYCLUSTER_PORT_DEPENDS ffmpeg boost poppler fcgi qt5-mac)
```

Comment:
no, poppler points to poppler-qt4-mac so looks like no pdf support on osx... Remove it or leave it in the hope that the port gets updated later?